libcurv/frag.cc

```diff
@@ -439,9 +440,22 @@ void export_frag_3d(
        "    return normalize(camera * vec3(pos, lens));\n"
        "}\n"
 
+       "float rand(vec2 co){\n"
+       "     return fract(sin(dot(co, vec2(12.9898, 78.233))) * 43758.5453);\n"
+       "}\n"
+
        "void mainImage( out vec4 fragColour, in vec2 fragCoord )\n"
        "{\n"
-       "    vec3 col = vec3(0.0);\n"
+       "const vec4 black = vec4(0,0,0,1.0);\n"
+       "vec2 uv = (fragCoord.xy  + vec2(iTime, 0)) / iResolution.xy;\n"
```

Comment:
This variable 'uv' appears to be used only inside the 'if (STOCH' if statement. Please move it inside the if statement.